docs/user-guide/test-zen-doc_Sean.md

```diff
@@ -53,50 +52,72 @@ FTP Port  |The FTP Port number for internal use. The default port is 21. If not
+Upload Zowe PAX for offline install | Upload a local (already downloaded) Zowe .PAX file using this option. <br />1. Click **UPLOAD PAX**.  <br />2. Select the downloaded PAX file and click **Open**.
+SMP/E | Use this option to install Zowe through SMP/E build outside the wizard. <br />1. Provide the location of the Runtime Directory. <br />2. Click **VALIDATE LOCATION**.                           <br />**Note:** In the case of SMP/E installation, in the Initialization window, under the **Installation** tab, confirm the dataset names used during installation. <br />3. Click **Save**. 
 
 
-12. Click **Continue to Component Installation**.
+2. Click **Continue to Component Installation**.
 
 ## Zowe Server Configuration
 
-Each page of the wizard from this point onward will prompt you for configuration values for a particular step of Zowe server configuration.
-Some pages may require an additional administrator with sufficient privileges to complete the step. If you encounter such a page, you can **Skip** the page to allow the step to be completed externally to the wizard. It is also possible to input the correct values on the page, and use the **View Job** button to collect the job content to provide to an administrator without submitting it.
-Each value entered in a page will be validated, and when you complete entry on the page, the following actions can be performed:
+Perform Zowe server configuration in the wizard by providing inputs to the prompts for configuration values.
+Some steps may require an administrator with sufficient privileges to complete the step. You have the option to **Skip** the page to allow the step to be completed externally to the wizard. It is also possible to input the correct values on the page, and click **View Job** to collect the job content to provide to an administrator without submitting it.
+Each value entered is validated.
+
+When you complete an entry the following actions can be performed:
+
+* **View/Edit YAML**  
+This feature lets you preview or adjust the YAML configuration that is used by Zowe. The prompts of the wizard are used to automatically generate the YAML contents, but you also have the options to review, edit, or import and export contents of the YAML file.
+* **View/Submit Job**  
+When you have completed providing the values for each step, you can click **View/Submit Job** to execute the operations of the configuration step upon the z/OS system. After a job is submitted, job output is available for review.
+
+* **Skip a step**  
+If you need to skip a step that you cannot perform such as an administrative seurity action that you cannot perform yourself, we recommend you click **View/Edit YAML**. This option allows you to copy the YAML in its current state. You can then provide the YAML and relevent JCL sample, and explanation for the particular step from this table to an administrator that is authorized to perform the task.
```

Comment:
redundant 1st sent

Reply:
Also please run this Grammarly. There's some spelling mistakes in this draft
*relevant